sru/csrc/main_test_cpp.cpp

```diff
@@ -0,0 +1,25 @@
+
+#include <torch/script.h> // One-stop header.
+
+#include <iostream>
+#include <memory>
+
+
+int main(int argc, const char* argv[]) {
```

Comment:
while this kind of test is useful, it would be good to have a test that checks that the output is equal when calling from python or c++. Not quite sure what the best way to do that is but seems important

Reply:
Yep. It's not easy to do so right now since the code has to be separately compiled thru `cmake` and `make`. Need to save and load a model artifact. I definitely wanna include some tests. Need some help since I'm not an expert (maybe Jing / speech modeling can help).

Reply:
what code needs to be compiled using `make`? dont suppose... could you add a README.md with full build instructions please?

Reply:
ok, I see. we first run cmake, then run make. forgot about this. so... what I would do is separte the build from the tests, like in the instructions, someone would run:

```
cmake
make
pytest
```

Reply:
like eg this is how it worked fro deepcl: https://github.com/hughperkins/DeepCL/blob/master/doc/Build.md

![Screen Shot 2020-08-25 at 11 58 52 AM](https://user-images.githubusercontent.com/20174596/91198304-64f85500-e6ca-11ea-960e-2d57f20ecd07.png)
![Screen Shot 2020-08-25 at 11 59 04 AM](https://user-images.githubusercontent.com/20174596/91198313-67f34580-e6ca-11ea-9b33-0c09aa6e4b54.png)